src/sematic-extra.css

```diff
@@ -23,4 +23,10 @@ button.tertiary {
 
 .ui.card.mentor-card:after {
   display: none;
-}
\ No newline at end of file
+}
+
+@media (min-width: 1025px) and (max-width: 1440px) {
```

Comment:
Why not doing this anyway (for other break points)? This problem also occurs for ~1092 px too.

Reply:
Just pushed again. Handled for 4k screens as well.